content/en/docs/instrumentation/python/exporters.md

```diff
@@ -136,71 +135,28 @@ OTLPSpanExporter(endpoint="<traces-endpoint>/v1/traces")
 
 There is not currently an OTLP/HTTP metric exporter.
 
-## Jaeger
+### Jaeger
 
 If you are using [Jaeger](https://www.jaegertracing.io/) to visualize trace
```

Comment:
Copy comment from above:

`If you are using [Jaeger](https://www.jaegertracing.io/) to visualize trace data, this is how to run it in a docker container:`

This sentence is confusing (you use jaeger but we ask you to install jaeger?)

I suggest something like:

`To try out the OTLP exporter, you can run  [Jaeger](https://www.jaegertracing.io/) as an OTLP endpoint and for trace visualization in a docker container`